config/v1/types_cluster_version.go

```diff
@@ -780,17 +803,30 @@ type ConditionalUpdate struct {
 	// +required
 	Release Release `json:"release"`
 
+	// riskNames represents names of ClusterVersionStatus.ConditionalUpdateRisks
+	// that are exposed to the release in a ConditionalUpdate.
+	// The cluster-version operator will evaluate these risks and only
+	// accept the update if there is at least one risk and for every risk
+	// it is either not applied to the cluster or considered acceptable
+	// by the cluster administrator.
+	// +kubebuilder:validation:items:MaxLength=256
+	// +kubebuilder:validation:MaxItems=100
+	// +listType=set
+	// +optional
+	RiskNames []string `json:"riskNames"`
+
 	// risks represents the range of issues associated with
 	// updating to the target release. The cluster-version
 	// operator will evaluate all entries, and only recommend the
 	// update if there is at least one entry and all entries
 	// recommend the update.
+	// DEPRECATED: the risks has been deprecated by RiskNames and will be removed in a future release.
```

Comment:
This [is a v1 API][1], which means [we can deprecate it, but cannot remove it until OCPv5][2].  Can we just put the period after `RiskNames`?  (and should it be `riskNames` in this line to match the JSON property, instead of the internal Go property?).

[1]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/api_overview/understanding-api-support-tiers#mapping-support-tiers-to-openshift-api-groups_understanding-api-tiers
[2]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/api_overview/understanding-api-support-tiers#api-tiers_understanding-api-tiers

Reply:
Done.